docs/cudf/source/index.rst

```diff
@@ -6,6 +6,7 @@ Welcome to cuDF's documentation!
    :caption: Contents:
 
    api.rst
+   basics.rst
```

Comment:
I think this should come after the 10min notebook(s), as that gives a broad introduction that most of our users are probably looking for first.